File: RealEstate.Domain/Roles/AdminUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RealEstate.Domain.Roles
{
    public class AdministrativeUser
    {
        public int Id { get; private set; }
        public string Name { get; private set; }
        public List<Appointment> Calendar { get; private set; } = new List<Appointment>();
    }

File: RealEstate.Presentation/RealEstateProject/RealEstateProject/src/app/models/roles/admin_user/admin-user.ts
import { Appointment } from "../appoiment/appoiment";

export class AdminUser {
  id: number;
  name?: string; 
  calendar: Appointment[] = [];

  constructor(id: number, name?: string, calendar?: Appointment[]) {
    this.id = id;
    this.name = name;
    this.calendar = calendar || []; 
  }
}
